backend/btrixcloud/uploads.py

```diff
@@ -190,7 +190,10 @@ async def _create_upload(
             self.event_webhook_ops.create_upload_finished_notification(crawl_id, org.id)
         )
 
-        asyncio.create_task(self.page_ops.add_crawl_pages_to_db_from_wacz(crawl_id))
+        await self.page_ops.add_crawl_pages_to_db_from_wacz(crawl_id)
```

Comment:
Do we not want to keep this async for uploads? Haven't tested the perf, maybe doesn't matter..

Reply:
Good point, for large uploads it's probably better to keep it async. Moved this and the collection updating to a new async task so that they'll complete in the right order.